core/src/test/scala/com/rackspace/com/papi/components/checker/GivenAWadlWithAuthenticatedByBase.scala

```diff
@@ -0,0 +1,301 @@
+    }
+    it should behave like accessIsNotAllowed("GET", "/methodId", InvalidAuthBy, desc)
+    it should behave like accessIsNotAllowed("GET", "/methodId", NoAuthBy, desc)
+
+    // reference a method with rax:authenticatedBy on parent+child resource and referencing method
+    desc = "method with a rax:authenticatedBy and referencing another method, parent+child resource rax:authenticatedBy"
+    Set("RSAKEY", "PASSWORD", "PASSCODE").foreach { authBy =>
+      it should behave like accessIsAllowed("GET", "/methodId/more", List(authBy), desc)
+    }
+    Set("APIKEY", "OTPPASSCODE", "IMPERSONATION", "FEDERATED").foreach { authBy =>
+      it should behave like accessIsNotAllowed("GET", "/methodId/more", List(authBy), desc)
+    }
+    it should behave like accessIsNotAllowed("GET", "/methodId/more", InvalidAuthBy, desc)
+    it should behave like accessIsNotAllowed("GET", "/methodId/more", NoAuthBy, desc)
+
+    // reference a method with rax:authenticatedBy on parent+child resource and referenced method
+    desc = "method and referencing another method with a rax:authenticatedBy, parent+child resource rax:authenticatedBy"
+    Set("RSAKEY", "PASSWORD", "FEDERATED").foreach { authBy =>
+      it should behave like accessIsAllowed("POST", "/methodId/more", List(authBy), desc)
+    }
+    Set("APIKEY", "PASSCODE", "OTPPASSCODE", "IMPERSONATION").foreach { authBy =>
+      it should behave like accessIsNotAllowed("POST", "/methodId/more", List(authBy), desc)
+    }
+    it should behave like accessIsNotAllowed("POST", "/methodId/more", InvalidAuthBy, desc)
+    it should behave like accessIsNotAllowed("POST", "/methodId/more", NoAuthBy, desc)
+
+    // reference a method with rax:authenticatedBy on parent+child resource and referencing+referenced method
+    desc = "method and referencing another method, both with a rax:authenticatedBy, and parent+child resource rax:authenticatedBy"
+    Set("RSAKEY", "PASSWORD", "IMPERSONATION").foreach { authBy =>
+      it should behave like accessIsAllowed("PUT", "/methodId/more", List(authBy), desc)
+    }
+    Set("APIKEY", "PASSCODE", "FEDERATED", "OTPPASSCODE").foreach { authBy =>
+      it should behave like accessIsNotAllowed("PUT", "/methodId/more", List(authBy), desc)
+    }
+    it should behave like accessIsNotAllowed("PUT", "/methodId/more", InvalidAuthBy, desc)
+    it should behave like accessIsNotAllowed("PUT", "/methodId/more", NoAuthBy, desc)
+  }
+
+  def createRequest(method: String, path: String, authBy: List[String]) =
+    base.request(method, path, DefaultMediaType, requestBody, parseContent = false, Map(AuthByHeader -> authBy))
+
+  def createConfig(useSaxon: Boolean): Config = {
+    val config = new Config
+
+    if (useSaxon) {
+      config.xsdEngine                    = "SaxonEE"
+    }
+    config.removeDups                     = true    // -d Wadl2Checker default is different from Config default.
```

Comment:
See other tests,  you typically want to iterate over these possible options.  Also you need to check that enableAuthenticatedByExtension also enables checkHeaders, etc.